types/rsocket-core/RSocketFrame.d.ts

```diff
@@ -99,14 +100,10 @@ export interface ErrorSource {
     /** The error string returned by the server. */
     message: string;
 }
-export class RSocketError extends Error {
-    /** Metadata about the error for use in introspecting at runtime. */
-    source: ErrorSource;
-}
 /**
  * Constructs an Error object given the contents of an error frame.
  */
-export function createErrorFromFrame(frame: ErrorFrame): RSocketError;
+export function createErrorFromFrame(frame: ErrorFrame): Error & { source: ErrorSource };
```

Comment:
Here, I was corrected by another engineer that here `RSocketError` wasn't a class but rather an `Error` which has a `source` field, so fixing it here as well.